README.md

```diff
@@ -294,6 +295,30 @@ const externalSchema = {
 const stringify = fastJson(schema, { schema: externalSchema })
 ```
 
+<a name="long"></a>
+#### Long integers
+Long integers (64-bit) are supported using the [long](https://github.com/dcodeIO/long.js) module.
+Example:
+```javascript
+const Long = require('long')
+
+const stringify = fastJson({
+  title: 'Example Schema',
+  type: 'object',
+  properties: {
+    id: {
+      type: 'integer'
```

Comment:
Yes, `integer` is part of the spec, I think @mcollina was talking about `long` numbers.